includes/namespace.php

```diff
@@ -30,48 +30,6 @@
 use WP_Error;
 use WP_Site;
 
-/**
- * Run logic to setup a new site with web stories.
- *
- * @since 1.2.0
- *
- * @return void
- */
-function setup_new_site() {
-	$injector = Services::get_injector();
-	if ( ! method_exists( $injector, 'make' ) ) {
-		return;
-	}
-
-	// Register web-story post type to setup rewrite rules.
-	$story = $injector->make( Story_Post_Type::class );
-	$story->register();
-
-	// Flush rewrite rules after registering post type.
-	rewrite_flush();
-
-	// Setup user capabilities.
-	$capabilities = $injector->make( User\Capabilities::class );
-	$capabilities->add_caps_to_roles();
-
-	// Not using Services::get(...) because the class is only registered on 'admin_init', which we might not be in here.
-	$database_upgrader = $injector->make( Database_Upgrader::class );
-	$database_upgrader->register();
-}
-
-/**
- * Flush rewrites.
- *
- * @since 1.7.0
- *
- * @return void
- */
-function rewrite_flush() {
```

Comment:
I don't understand why remove it? Just saves lots of `phpcs:ignore`

Reply:
I actually updated the PHPCS config because those PHPCS warnings are really false positives, so all those `phpcs:ignore` comments are not needed anymore.